offchain-modules/packages/x/src/core.ts

```diff
@@ -45,6 +58,7 @@ export class ForceBridgeCore {
   private static _ckb: CKB;
   private static _ckbIndexer: CkbIndexer;
   private static _keystore: KeyStore;
+  private static _xChainHandler: XChainHandlers;
```

Comment:
i dont think the handler should be in the `ForceBridgeCore`, `cli, and `rpc` also depend on `ForceBridgeCore`, but they don't need the handler

Reply:
Handler is a common module,  all of collector, rpc and signer server need handler